internal/databases/postgres/backup_push_handler.go

```diff
@@ -101,6 +101,8 @@ type BackupHandler struct {
 	pgInfo         BackupPgInfo
 }
 
+var isSimpleComposerTarBallType bool
```

Comment:
Using global variables should be avoided, pass this setting directly to the `NewTarballStreamer`